pages/index.js

```diff
@@ -1,23 +1,26 @@
 import { v4 as uuidv4 } from "uuid";
+import Link from "next/link";
 
-import EventCard from "./components/EventCard";
-import Flex from "./components/Flex";
-import { events } from "./api/db.js";
+import EventCard from "../components/EventCard";
+import Flex from "@/components/StyledComponents/Flex";
 
-export default function HomePage() {
+export default function HomePage({ events }) {
   return (
-    <Flex $direction="column">
-      {events.map((event) => {
-        return (
-          <EventCard
-            key={uuidv4()}
-            title={event.title}
-            date={event.date}
-            type={event.type}
-            desc={event.desc}
-          />
-        );
-      })}
-    </Flex>
+    <>
+      <Link href="/create">Event Form</Link>
+      <Flex $direction="column">
+        {events.map((event) => {
+          return (
+            <EventCard
+              key={uuidv4()}
```

Comment:
Here you better should pass also an `event.id` prop here and assign your id via `uuidv4` inside the `addEvent` function like: 

```
  function addEvent(newEvent) {
    setEvents([...events, { ...newEvent, id: uuidv4() }]); 
  }
```